test/utils/mocha-helpers.js

```diff
@@ -2,6 +2,7 @@ const chai = require("chai");
 const sinon = require("sinon");
 const sinonChai = require("sinon-chai");
 const chaiAsPromised = require("chai-as-promised");
+process.env.ENABLE_PREVIEW = "test";
```

Comment:
For local running we set this to 'development' in a .env file, but that causes tests to fail as we are then testing the development-only router.

This will now ensure that local tests properly replicate the pipeline runners